### ISSUE TITLE
Doesn't seem to work on Chrome OS

### ISSUE THREAD

**Issue by DRschulman** [GitMurf/roam-highlighter#57]
On my Pixelbook Go, This is what I get when I select settings; and then try to click on any option:
![Screenshot 2020-08-25 at 8 01 37 PM](https://user-images.githubusercontent.com/45397647/91240060-ee317b00-e70e-11ea-879c-ecb997439dcd.png)

I can't select or change any options?

**Comment by GitMurf** [comment]
@DRschulman am I seeing that correctly.... you are seeing that giant blue checkbox in the middle of the screen when you activate the highlighter?

can you please try activating the highlighter on my Github page at: https://github.com/GitMurf/roam-highlighter

I want to see if it has to do with your PixelBook Go or if it has to do with the website you were visiting because sometimes that happens if they have funky HTML / CSS.

**Comment by DRschulman** [comment]
Thanks GM - it must have been the website, as I can't replicate it. And, since it hasn't recurred, I had obviously forgotten about it until you responded. BTW - you may have gotten an influx of RH users since last Wed, as I sang it's praises on the "Believers" call. Bardia wasn't aware of it - but knew of you, and shared the link to the RH  extension with the crowd.

**Comment by GitMurf** [comment]
@DRschulman thanks for the plug! I wish I had time these days to implement some of the enhancements I've had on my radar for a while now! But I have been pretty satisfied with it being in a pretty stable/reliable state outside of some of the corner cases issues that occasionally pop up. 

There is another Roam Highlighter extension that is much prettier than mine by Daniel Wirtz. He may even be a Believer too. Did that come up on the call? I still am interested in people's feedback that have used both highlighters to see how mine compares or how I can improve mine. Have you used them both?

**Comment by DRschulman** [comment]
Now I'm really embarrassed. I just checked - I'm using Wirtz's, and that's
who I plugged at the believer's meeting. So sorry. That's why I couldn't
replicate the problem, I'm not using your extension. I had a lot of issues
with yours - couldn't adjust the settings, couldn't get it to work.
Switched to Wirtz's; no problems. Then I forgot about it. That's who Bardia
knew - he may know you too, but now that you reminded me of Wirtz's name, I
went to the extension page, and that's the one I have loaded. Oops. It's
been a month since I was playing with these extensions, but have been using
Wirtz's steadily since. - which is why I forgot about the issues. So sorry.

- David Schulman
